crates/fj-kernel/src/objects/full/solid.rs

```diff
@@ -5,7 +5,9 @@ use crate::{
     storage::Handle,
 };
 
-/// A 3-dimensional shape
+/// A 3-dimensional shape, built from [`Shell`]s. Many Solids will contains only
+/// one shell, but if the Solid contains cavities they will be reprented by a
+/// shell each, as well as a shell for the outside.
```

Comment:
A typo:
```suggestion
/// A 3-dimensional shape, built from [`Shell`]s. Many Solids will contains only
/// one shell, but if the Solid contains cavities they will be represented by a
/// shell each, as well as a shell for the outside.
```

Additional comments:
- I think it's better to have a short 1-line header for a doc comment, before going into details. I believe (although I haven't checked in a while) that leads to nicer generated documentation, than starting with a multi-line paragraph.
- This leaves out the possibility that the `Solid` could be made up of multiple bodies, i.e. have multiple exterior shells.